pkg/volume/volume.go

```diff
@@ -171,6 +177,29 @@ type Detacher interface {
 	UnmountDevice(deviceMountPath string) error
 }
 
+// NewTryAgainError returns a new instance of TryAgainError error.
+func NewTryAgainError(message string) error {
```

Comment:
nit: I'm going to be pedantic about naming here--errors should describe what happened, not what the handlers should do. Consumers that handle the error can decide what to do based on what the error was.